### ISSUE TITLE
feature request: support for 2205A

### ISSUE THREAD

**Issue by Vbitz** [azonenberg/scopehal-pico-bridge#23]
I'd like to use ngscopeclient with my Pico 2205A.

Looks like the scope needs to use the old ps2000 driver which has a different API compared to the 3000A and 6000A series scopes.

I'm working on implementing this but I'll need to write a separate version of the bridge to support this. Is this a good approach or am I barking up the wrong tree?

**Comment by azonenberg** [comment]
Yes, that is correct. You'll probably also need to add some logic to the PicoOscilloscope class in ngscopeclient.

It's annoying but this is really the only way, because Pico doesn't have a single SDK for all of their scopes. They have separate SDKs, with similarly named and configured functions, for each product line and you really just have to work around that.

**Comment by mdouchement** [comment]
Hello, I'm quite interested in the support of this hardware.
Is there any news or public PoC?

**Comment by Vbitz** [comment]
I had a go at implementing this using the older API but it needs almost a complete rewrite compared to the existing Pico driver.

I haven't had time to implement that unfortunately.
